doc/api/util.md

```diff
@@ -14,6 +14,60 @@ it:
 const util = require('node:util');
 ```
 
+## `util.addSafeAbortSignalAbortListener(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `signal` {AbortSignal}
+* `listener` {Function|EventListener}
+* `options` {Object}
+  * `passive` {boolean} When `true`, serves as a hint that the listener will
+    not call the `Event` object's `preventDefault()` method.
+    **Default:** `false`.
+  * `signal` {AbortSignal} The listener will be removed when the given
+    AbortSignal object's `abort()` method is called.
+* Returns: {Disposable} that removes the `abort` listener.
+
+Listens once to the `abort` event on the provided `signal`.
+The listeners will be triggered even if the `abort` event's propagation has
+been stopped.
+
+```cjs
+const { addSafeAbortSignalAbortListener } = require('node:util');
+
+function example(signal) {
+  let disposable;
+  try {
+    signal.addEventListener('abort', (e) => e.stopImmediatePropagation());
+    disposable = addSafeAbortSignalAbortListener(signal, (e) => {
+      // Do something when signal is aborted.
+    });
+  } finally {
+    disposable?.[Symbol.dispose]();
+  }
```

Comment:
I assume we don't want to have an example using the `using` keyword before it is natively supported 🤔